modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less

```diff
@@ -108,21 +108,29 @@
         fill: @accessibility-issue-warn-heading-text-color;
       }
 
-      a .tox-icon {
-        color: @accessibility-issue-warn-heading-text-color;
+      /* stylelint-disable-next-line no-descending-specificity */
```

Comment:
Should we just nest it inside `a.tox-button--naked` so that we don't have a disable a linting rule. Is this really that much easier to parse/read? Is that what the rule is complaining about?

Reply:
I think it doesn't complain about `info` because it's first. If you swapped the order and put something else first then `info` would get the warning too.

Likely the only way to work around the warning is specify the 3 button colours and then repeat the 3 button selectors to set hover/focus/active later in the file. Given we know `error`/`warning`/`info` won't be used at the same time I guess suppressing the warning is fine.

Reply:
Fair enough. Withdrawn.